sdk/scheduler/src/main/java/com/mesosphere/sdk/storage/PersisterUtils.java

```diff
@@ -190,4 +201,68 @@ public static void clearAllData(Persister persister) throws PersisterException {
             }
         }
     }
+
+    public static void backUpFrameworkZKData(Persister persister) throws PersisterException {
+        try {
+            persister.recursiveDelete(BACKUP_ROOT_NAME);
```

Comment:
Could put the backup data in a timestamped subdir, like `backup/2018-07-18-095146/` (or some other format). Then there's much less risk of conflict with prior backups, and it gives operators some indication of where the backup could have come from.

Could see us wanting to have backups like this in the future, so it doesn't hurt to figure out a schema for this now.